src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.RuntimeIdentifierInference.targets

```diff
@@ -122,7 +122,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     compiling the exe if no copy-local native dependencies were resolved from NuGet 
   -->
   <Target Name="AdjustDefaultPlatformTargetForNetFrameworkExeWithNoNativeCopyLocalItems"
-          DependsOnTargets="ResolvePackageDependenciesForBuild"
+          AfterTargets="ResolvePackageDependenciesForBuild"
```

Comment:
I am trying to make sure that this is done as soon as we know the right PlatformTarget. Somebody could indeed force it to be sooner.